forge-gui/res/cardsfolder/upcoming/magus_lucea_kane.txt

```diff
@@ -0,0 +1,13 @@
+Name:Magus Lucea Kane
+ManaCost:1 G U R
+Types:Legendary Creature Human Tyranid Wizard
+PT:1/1
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ At the beginning of combat on your turn, put a +1/+1 counter on target creature.
+SVar:TrigPutCounter:DB$ PutCounter | ValidTgts$ Creature | CounterType$ P1P1 | CounterNum$ 1
+A:AB$ Mana | PrecostDesc$ Psychic Stimulus - | SubAbility$ DBEffect | Cost$ T | Produced$ C | Amount$ 2 | SpellDescription$ Add {C}{C}. When you next cast a spell with {X} in its mana cost or activate an ability with {X} in its activation cost this turn, copy that spell or ability. You may choose new targets for the copy. (A copy of a permanent spell becomes a token.)
+SVar:DBEffect:DB$ Effect | Triggers$ CopySpellCast,CopyAbility | ExileOnMoved$ Exile
```

Comment:
ExileOnMoved is for Remembered
but you can put `OneOff$ True` on the trigger instead

Reply:
"OneOff$ True on the trigger"

you put it on the Effect